### PR TITLE
Add translations json file for bulgarian (bg)

### PR DESCRIPTION
**Pull request by lachezar-gizdov**
translate from the en.json file all the values to Bulgarian (BG).

### CODE REVIEW

File: custom_components/petkit/translations/bg.json [hunk 1]
```diff
@@ -0,0 +1,743 @@
...
+      "reauth_successful": "Повторната автентификация беше успешна"
+    },
+    "error": {
+      "cannot_connect": "Неуспешно свързване",
+      "invalid_auth": "Невалидна автентификация. Правилни ли са вашите данни за вход?",
+      "no_devices": "Не са открити устройства на акаунта",
+      "server_busy": "Сървърите на PetKit са заети. Моля, опитайте отново по-късно.",
+      "petkit_error": "Възникна неизвестна грешка. Моля, вижте логовете на Home Assistant за повече подробности.",
+      "region_error": "Моля, изберете държавата, свързана с вашия акаунт.",
+      "timezone_error": "Часовата зона не може да бъде намерена. Ако използвате Home Assistant като самостоятелен Docker контейнер, трябва да дефинирате променливата TZ."
```
Comment (RobertD502) [inline custom_components/petkit/translations/bg.json:14]: Please look at the current en.json file, the timezone_error was updated before the last release.

File: custom_components/petkit/translations/bg.json [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+    },
+    "step": {
+      "user": {
+        "data": {
+          "email": "Имейл на акаунта в PetKit",
+          "password": "Парола на акаунта в PetKit",
+          "region": "Изберете държавата, свързана с вашия акаунт",
+          "polling_interval": "Интервал на опитване (секунди)"
```
Comment (RobertD502) [inline custom_components/petkit/translations/bg.json:22]: Please look at the current en.json file, a `timezone` key exists here now.

File: custom_components/petkit/translations/bg.json [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+        },
+        "title": "Попълнете вашите данни за PetKit"
+      },
+      "reauth_confirm": {
+        "data": {
+          "email": "Имейл на акаунта в PetKit",
+          "password": "Парола на акаунта в PetKit",
+          "region": "Изберете държавата, свързана с вашия акаунт",
+          "polling_interval": "Интервал на опитване (секунди)"
```
Comment (RobertD502) [inline custom_components/petkit/translations/bg.json:31]: Please look at the current en.json file, a `timezone` key exists here now.

File: custom_components/petkit/translations/bg.json [hunk 4]
```diff
@@ -0,0 +1,743 @@
...
+        "title": "Повторно удостоверяване с вашите данни за PetKit"
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "petkit_options": {
+        "data": {
+          "region": "Изберете държавата, свързана с вашия акаунт",
+          "polling_interval": "Интервал на опитване (секунди)"
```
Comment (RobertD502) [inline custom_components/petkit/translations/bg.json:42]: Please look at the current en.json file, a `timezone` key exists here now.

File: custom_components/petkit/translations/bg.json [hunk 5]
```diff
@@ -0,0 +1,743 @@
...
+        "name": "Планиран контейнер за дозиране 1"
+      },
+      "planned_dispensed_hopp_two": {
+        "name": "Планиран контейнер за дозиране 2"
+      },
+      "dispensed_hopp_one": {
+        "name": "Дозиран контейнер 1"
+      },
+      "dispensed_hopp_two": {
+        "name": "Dispensed hopper 2"
```
Comment (RobertD502) [inline custom_components/petkit/translations/bg.json:651]: Did you miss this translation?

**Review by RobertD502 — CHANGES_REQUESTED** [review]

File: custom_components/petkit/translations/bg.json (hunk 5 above)
Comment (lachezar-gizdov) [reply custom_components/petkit/translations/bg.json]: yes, fixed.

**Review by RobertD502 — APPROVED** [review]
Thanks!

**Pull request merged**